org.chromium.Chromium.yaml

```diff
@@ -32,6 +32,14 @@ add-extensions:
     bundle: true
     autodelete: true
 
+  org.chromium.Chromium.UnifiedExtension:
```

Comment:
Hm, wouldn't it be cleaner for the `.Extension` extension point to assume the role of `.UnifiedExtension`, instead of adding a new one?

Reply:
My main concern was conceptual, it felt confusing to have `.Policy` and `.NativeMessaingHost` be specific but `.Extension` is "unified".

That being said, the others are essentially "legacy", so maybe this isn't that major. I'll ask around for some other feedback as well.

Reply:
>it felt confusing to have `.Policy` and `.NativeMessaingHost` be specific but `.Extension` is "unified"

Well, maybe name the new extension point something different altogether to avoid confusion, like `.Addons`?

Reply:
In Endless OS we use the current extension points to magic configuration in from the host system, [namely](https://github.com/endlessm/eos-boot-helper/blob/eos3.9/tmpfiles.d/chromium-system-services.conf.in) policies from `/etc/chromium-browser` and extensions from `/usr/share/chromium/extensions`.  (For the curious, we specifically use this to [conditionally enable Cisco OpenDNS Family Shield](https://github.com/endlessm/eos-theme/tree/master/safe-defaults), and [install AdBlock Plus by default](https://github.com/endlessm/eos-browser-tools/blob/master/adblock-plus/cfhdojbkjhnklbpkdaibdccddilifddb.json), respectively. We also have some installations which use custom mandatory policies installed to `/etc/chromium-browser`.)

Because this is done not with 'real' Flatpaks but with configuration that is shipped in the OS itself, we cannot easily update it in lockstep with a new Chromium Flatpak. (We have a mechanism to trigger an app update along with an OS update, but not the other direction.)

So,

> Hm, wouldn't it be cleaner for the `.Extension` extension point to assume the role of `.UnifiedExtension`, instead of adding a new one?

I don't have any strong feelings on the aesthetics of this versus the other suggested names, but, is it possible to do this without breaking Endless OS's existing use of this extension point? Otherwise:

> That being said, the others are essentially "legacy", 

They may be legacy but we really need them to keep working :)

Reply:
Or if it's not possible to keep them working, we need to be able to coordinate an OS update for at least best-effort adjustment to the change.

Reply:
I guess existing extensions will keep working as before if the `.Extension` extension point starts merging not only `extensions` dir but also `policies` and `native-messaging-hosts` dirs. Right?

Reply:
> is it possible to do this without breaking Endless OS's existing use of this extension point?

That was pretty much the goal of the symlinks (that apparently aren't working). The old ones are annotated as "legacy" to try to discourage usage (well, they will be when I get around to updating the README I guess), but they'll definitely continue to work.

Also yeah re-using `.Extension` should not be a breaking change (are there even any users of this yet?). For me, it's just the inconsistency of having two extensions based on ways we can extend Chromium (`.Extension` and `.Policy`) but one is actually all-encompassing. Though, I guess, if Endless will probably end up being the only user of `.Policy`, all we'd need is a comment for clarification then?

Reply:
I suggest reusing `.Extension` and documenting that the others are legacy.

If we're prepared to ignore non-Endless OS users of the other extension points (indeed, per #78 they are broken anyway) then this also dodges the "tmpfs doesn't exist if there aren't any runtimes mounted there" issue. On Endless OS it is guaranteed that `org.chromium.Chromium.Extension.system-extensions=local` always exists.